core/src/main/scala/com.snowplowanalytics.stream/loader/Config.scala

```diff
@@ -275,4 +273,9 @@ object Config {
     failureStrings.mkString("\n")
   }
 
+  private def getKinesisEndpoint(region: String, customEndpoint: Option[String]): String =
+    customEndpoint.getOrElse(region match {
+      case cn @ "cn-north-1" => s"https://kinesis.$cn.amazonaws.com.cn"
```

Comment:
In the collector we recently [added cn-northwest-1 to this list](https://github.com/snowplow/stream-collector/blob/47bd03c22235b4f70c0d752ad289b25fc40e414b/core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/model.scala#L118).  See [this collector issue](https://github.com/snowplow/stream-collector/issues/152) from an open source user.

I know it's not related to the current issue, but you could add a new issue.